gateway/src/apicast/backend_client.lua

```diff
@@ -120,6 +121,10 @@ local function auth_path(using_oauth)
          '/transactions/authorize.xml'
 end
 
+local function report_path()
```

Comment:
This could be just module property instead of a function, right?

Reply:
Right :+1: 
Fixed.